oak_containers_launcher/src/qemu.rs

```diff
@@ -109,6 +114,9 @@ impl Qemu {
         if let Some(memory_size) = params.memory_size {
             cmd.args(["-m", &memory_size]);
         };
+        if let Some(num_cpus) = params.num_cpus {
```

Comment:
Ah, I see now why you have Some, but I'd assume it would work with 1 too and you could drop the if?

Reply:
Done; I've changed it to use a default value of 1.